tests/auth/test_backends.py

```diff
@@ -79,52 +22,3 @@ def test_extract_email_from_clientid(client_id, exp_email, exp_exception):
     else:
         email = tca._extract_email_from_clientid(client_id)
         assert email == exp_email
-
-
-@pytest.mark.django_db
-@pytest.mark.parametrize(
-    ('result', 'exp_username', 'email', 'exp_create_user'),
-    [({'status': 'auth-success',
-       'scopes': [],
-       'clientId': 'email/user@foo.com'},
-      'email/user@foo.com',
-      'user@foo.com',
-      True),
-     ({'status': 'auth-success',
-       'scopes': [],
-       'clientId': 'email/emailaddressexceeding30chars@foo.com'},
-      'email/emailaddressexceeding30chars@foo.com',
-      'emailaddressexceeding30chars@foo.com',
-      True),
-     ({'status': 'auth-success',
-       'scopes': ["assume:mozilla-user:foo@bar.net"],
-       'clientId': 'email/foo@bar.net'},
-      'email/foo@bar.net',
-      'foo@bar.net',
-      False),
-     ])
-def test_existing_email_create_user(test_user, monkeypatch, result,
```

Comment:
This is also still a valid test.  It just needs some tweaking in this line:

        existing_user = User.objects.create(username="email/foo@bar.net", email=email)

And the comment should be changed because it's no longer relying on the scope for anything, so you can remove the part about matching and migrating the username.